setup.cfg

```diff
@@ -1,2 +1,53 @@
 [metadata]
-description-file = README.rst
+classifiers =
+    License :: OSI Approved :: Apache Software License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+license = Apache 2.0
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+name = dwave-ocean-sdk
+project_urls =
+    Changes = https://github.com/dwavesystems/dwave-ocean-sdk/releases
+    Documentation = https://docs.ocean.dwavesys.com
+    Souce Code = https://github.com/dwavesystems/dwave-ocean-sdk
+url = https://github.com/dwavesystems/dwave-ocean-sdk
+version = attr: dwaveoceansdk.__version__
+
+# keep description, author and author email up to date with dwaveoceansdk.package_info
+author = D-Wave Systems
+author_email = tools@dwavesys.com
+description = Software development kit for open source D-Wave tools
+
+[options]
+install_requires =
+    dimod==0.10.9
+    dwave-cloud-client==0.9.2
+    dwave-greedy==0.2.1
+    dwave-hybrid==0.6.4
+    dwave-inspector==0.2.8
+    dwave-neal==0.5.8
+    dwave-networkx==0.8.10
+    dwave-preprocessing==0.3.1.post0
+    dwave-qbsolv==0.3.3
+    dwave-system==1.10.0
+    dwave-tabu==0.4.2
+    dwavebinarycsp==0.1.3
+    minorminer==0.2.6
+    penaltymodel-cache==0.4.4
+    penaltymodel-maxgap==0.5.5; platform_machine != "x86_64" and platform_machine != "amd64" and platform_machine != "AMD64"
+    penaltymodel-mip==0.2.5; platform_machine == "x86_64" or platform_machine == "amd64" or platform_machine == "AMD64"
```

Comment:
Do you happen to know which version of pip/setuptools this requires?

Reply:
I don't but I'll add https://github.com/dwavesystems/dimod/blob/24e06a82b0caf7128a5bc92ae758863359947bf6/pyproject.toml#L3 since that's required by our dependencies anyway.

Reply:
Looks like [36.2.0](https://setuptools.pypa.io/en/latest/history.html#v36-2-0), Jul 2017.

Reply:
Thanks for checking.

Reply:
Just kidding, [36.2.7](https://setuptools.pypa.io/en/latest/history.html#v36-2-7), August 2017